core/storage/classifier/gae_models.py

```diff
@@ -160,7 +160,8 @@ def query_new_and_pending_training_jobs(cls, cursor=None):
                 datastore cursor.
 
         Returns:
-            List of the ClassifierTrainingJobModels with status new or pending.
+            list. List of the ClassifierTrainingJobModels with status new or
```

Comment:
list(ClassifierTrainingJobModel).

In general, if the typeinfo contains `list`, please ensure that it is `list(...)` instead.

Reply:
Done.